Azure Function Support as feature (#44)
kedacore/keda-docs: content/_index/features/azure-functions.md

+++
title = "Azure Functions Support"
weight = 60

[asset]
  icon = "fab fa-microsoft"
+++

> Run & scale your Azure Functions on Kubernetes with production support.

Reviewer: ```suggestion
Run & scale your Azure Functions on Kubernetes in production workloads.
```

We don’t have a way yet you could open a support ticket and get production “Azure support” for Kubernetes based functions so renaming to be clear we endorse for production but don’t have a support plan
